api/utils/get_transaction.py

```diff
@@ -0,0 +1,36 @@
+from flask_restful import Resource
+from flask import request
+from config import app
+import iota
+import json
+
+
+class Get_transaction (Resource):
+    def get(self, tx_hash):
+        if len(tx_hash) != 81:
+            return {
+                'message': 'Type Not Included!'
+            }, 400
+
+        api = iota.Iota(app.config['API_URI'])
+
+        Tx = tx_hash
+        try:
+            data = api.get_trytes(hashes=[Tx])
```

Comment:
From line 19 to 27, this may be a utility function and put it in `api/utils` ?

Reply:
Fix in f71aa2f.